### ISSUE TITLE
Panel rendering/sharing: Grafana always renders home dashboard

### ISSUE THREAD

**Issue by dpsenner** [grafana/grafana#22948]
**What happened**:

Alert notifications began to include the home dashboard instead of the panel that caused the alert. Attempting to render some random panel always renders the home dashboard, too.

**What you expected to happen**:

Grafana should render the panel instead of the home dashboard.

**How to reproduce it (as minimally and precisely as possible)**:

1. Start grafana server,
1. Open a dashboard
1. Share a panel
1. Direct link rendered image

**Anything else we need to know?**:

This used to work fine some weeks ago. It may be a regression or a modification in the system. I used to render images using the deprecated built-in option. The problem has presented itself back then. Only then I attempted to install grafana-image-renderer and use the plugin to render. The symptom is still the same. It appears like this is not related to rendering itself but may be some url rewrite? The grafana server is installed in a virtual server (virtualbox) which runs apache2 and the site is configured to do ProxyPass / ProxyPassReverse.

**Environment**:
- Grafana version: 6.7.1
- Data source type & version: postgresql
- OS Grafana is installed on: ubuntu 18.04.1 lts
- User OS & Browser: ubuntu & chrome
- Grafana plugins:
  - grafana-clock-panel @ 1.0.3
  - grafana-image-renderer @ 1.0.11
  - grafana-polystat-panel @ 1.1.0
  - gretamosa-topology-panel @ 1.0.0
  - jdbranham-diagram-panel @ 1.6.1
- Others:
  - N/A

**Comment by dprokop** [comment]
@marefr do you mind taking a look at this? Couldn't reproduce, and I can't see anything in the front-end that could cause this issue

**Comment by lasertiger** [comment]
Same here - same version of Grafana AND also set up with Proxy Pass.

It always renders a portion of the home dashboard only.

**Comment by marefr** [comment]
@dpsenner @lasertiger make sure you have at least v1.0.11 of image renderer plugin installed. Also try latest Grafana stable version. 

Set environment variable so it's available for Grafana process, e.g. `export GF_RENDERER_PLUGIN_VERBOSE_LOGGING=true` 

[Then enable rendering debug logging in Grafana](https://grafana.com/docs/grafana/latest/administration/image_rendering/#troubleshoot-image-rendering)

Please report Grafana server log debug output here when rendering an image. Also, please include what type of panel (plugin) you're trying to render if that makes any difference and possibly include dashboard/panel json model.

**Comment by dpsenner** [comment]
It appears like grafana-image-renderer is already at the latest version available:

```bash
$ sudo grafana-cli plugins ls
installed plugins:
grafana-clock-panel @ 1.0.3
grafana-image-renderer @ 1.0.11
grafana-polystat-panel @ 1.1.0
gretamosa-topology-panel @ 1.0.0
jdbranham-diagram-panel @ 1.6.1
```
I'll need some hours of spare time to investigate retrieving the logfiles.

**Comment by marefr** [comment]
Are you maybe using `serve_from_sub_path` setting? In that case duplicate of #21925

**Comment by dpsenner** [comment]
One shot into the dark.. hit!

```bash
~$ sudo grep serve_from_sub_path /etc/grafana/grafana.ini
serve_from_sub_path = true
```

What is going to be the version that the fix is shipped with?

**Comment by kevinisbest** [comment]
Hi @marefr, I set the `serve_from_sub_path = true` and Grafana always renders the home dashboard.
And I refer [this reply](https://github.com/grafana/grafana/issues/21925#issuecomment-614419729) set the `serve_from_sub_path = false` but the entire dashboard broken
![image](https://user-images.githubusercontent.com/10130649/83723544-76a10200-a671-11ea-8491-3a4255feb1ec.png)

**Comment by marefr** [comment]
Duplicate of #21925

**Comment by marefr** [comment]
Think this is a duplicate and fix is included in Grafana v7.0. Please try and upgrade to Grafana v7.0.3 and let us know the outcome.

**Comment by kevinisbest** [comment]
Yes, I update grafana-server to 7.0.3 and grafana-image-renderer to 2.0.0
